srcpkgs/arturo/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'arturo'
+pkgname=arturo
+version=0.9.83
+revision=1
+archs="x86_64"
+hostmakedepends="choosenim xz"
+short_desc="Simple, expressive & portable programming language"
+maintainer="Eloi Torrents <eloitor@duck.com>"
+license="MIT"
+homepage="http://arturo-lang.io/"
+distfiles="https://github.com/arturo-lang/arturo/archive/refs/tags/v${version}.tar.gz"
+checksum=0bb3632f21a1556167fdcb82170c29665350beb44f15b4666b4e22a23c2063cf
+
+do_build() {
+	choosenim 1.6.12 -y
```

Comment:
```suggestion
_nim_version="1.6.12"

do_build() {
	choosenim ${_nim_version} -y
```

Reply:
we probably shouldn't be using choosenim at all... we don't use rustup in rust templates